### DISCUSSION TITLE
No messages prior to certain date

### DISCUSSION THREAD

**Discussion by ciction** [Tyrrrz/DiscordChatExporter#968] [Q&A]
I am using the CLI exporter on Linux.
Lately my exports don't seem to cover the given date range.

Is it possible that the discord API is limiting how far you can go back in time? (or based on  message count)

**Comment by Tyrrrz** [comment]
Are you using a bot by any chance?

**Reply by ciction** [reply]
I actually do have a discord chatbot that I created as a test if that's what you're asking, but this was just for taking a one-time private message export. Is there a relation between the two?
By the way, exporting the entire timeline seems to work. It's only combining --"before" and "--after" that seems to cause the issue.
Maybe i should create a bug report under "issues" ?

**Reply by Tyrrrz** [reply]
No, I was asking because bot permissions are more granular so it's possible you have access to read the channel, but not its history. I'm not sure if the same can be applied to users too. But that wouldn't explain why the messages are visible in the client (as I assume, they are?).

**Reply by Tyrrrz** [reply]
Maybe you're combining "before" and "after" incorrectly? What values did you use?
